opensuse-tomcat-jul/pom.xml

```
@@ -0,0 +1,146 @@
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>start-docker-image</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>stop-docker-image</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <filter>opensuse-tomcat-jul</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <images>
+                        <image>
+                            <alias>opensuse-tomcat-jul</alias>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion}</name>
+                            <build>
+                                <dockerFileDir>.</dockerFileDir>
+                                 <assembly>
+                                    <basedir>/</basedir>
+                                    <inline>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <outputDirectory>/lib</outputDirectory>
+                                                <useTransitiveDependencies>false</useTransitiveDependencies>
+                                            </dependencySet>
+                                        </dependencySets>
+                                    </inline>
+                                </assembly>
+                                <args>
+                                    <!-- Enable internet access -->
+                                    <http_proxy>${env.HTTP_PROXY}</http_proxy>
+                                    <https_proxy>${env.HTTPS_PROXY}</https_proxy>
+                                    <no_proxy>${env.NO_PROXY}</no_proxy>
+                                </args>
+                            </build>
+                        </image>
+                        <image>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat-test${dockerProjectVersion}</name>
+                            <build>
+                                <from>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion</from>
```

```suggestion
                                <from>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion}</from>
```
Missing closing } bracket.